app/services/member-consequences.js

```diff
@@ -349,6 +349,8 @@ export default Ember.Service.extend({
         default:
           break;
       }
+
+      tag.save();
```

Comment:
This MUST check for the `[null]` value and the tag's answer before saving, because that represents a redacted question and MUST NOT be sent back to Solarium. Use a similar implementation as was in `saveTags()` question route action, but do the check in the tag adapter's `sendTag()`.